doc/api/cli.md

```diff
@@ -1454,6 +1454,16 @@ Starts the Node.js command line test runner. This flag cannot be combined with
 See the documentation on [running tests from the command line][]
 for more details.
 
+### `--test-bail`
+
+<!-- YAML
+added:
+  - REPLACEME
+-->
+
```

Comment:
Can we mark it as experimental?

Reply:
Why experimental? I dont think the design of this feature is quite simple

Reply:
Why not? Having it experimental lets us ship breaking change if we deemed necessary, and I think it has no downsides

Reply:
Marked as experimental